doc/DeveloperGuide.md

```diff
@@ -39,7 +48,8 @@
 
 ### JUnit tests
 
-* In Eclipse, right-click on the `test/java` folder and choose `Run as` > `JUnit Test`
+* In IntelliJ, right-click on the `test` folder and choose `Run 'All Tests'`
+* If `StorageFileTest` throws `NoSuchFileException`, click the `All in test` drop down menu at the top right corner of the window. Select `Edit configurations...`. Under `Configuration`, change the `Working directory` to the `addressbook-level2` folder
```

Comment:
Should be a separate info box on its own (reason: it is only done once and it is "additional" information).